px-data-grid-date-renderer.html

```diff
@@ -1,17 +1,35 @@
 <link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../px-tooltip/px-tooltip.html">
 <link rel="import" href="px-data-grid-renderer-mixin.html">
+<link rel="import" href="px-data-grid-theme.html">
 
 <dom-module id="px-data-grid-date-renderer">
   <template>
-    <style>
+    <style include="px-data-grid-styles">
       :host {
         display: flex;
         align-items: center;
+        flex-grow: 1;
+      }
+
+      #editingTemplate {
+        width: 100%;
+      }
+
+      input {
+        box-sizing: border-box;
       }
     </style>
     <div id="value">[[_getFormattedDate(value)]]</div>
     <div id="editingTemplate">
-      <input type="date" value="{{value::change}}">
+      <input id="editingTemplateInput" class$="{{getClasses(validationResult)}}" type="date" value="{{value::change}}">
+      <px-tooltip
+        for="editingTemplateInput"
+        orientation="top"
+        tooltip-message="[[validationResult.message]]"
+        opened="[[!validationResult.valid]]"
+        ignore-target-events="true">
```

Comment:
minor thing, but is polymer way to do this just to define ignore-target-events attribute without value